base/errorshow.jl

```diff
@@ -165,6 +165,42 @@ end
 
 typesof(args...) = Tuple{Any[ Core.Typeof(a) for a in args ]...}
 
+function print_with_compare(io::IO, a::DataType, b::DataType, color)
+    if a.name === b.name
+        Base.show_type_name(io, a.name)
+        n = min(length(a.parameters), length(b.parameters))
+        print(io, '{')
+        for i = 1:n
+            print_with_compare(io::IO, a.parameters[i], b.parameters[i], color)
+            i < n && print(io, ',')
+        end
+        print(io, '}')
+    else
+        printstyled(io, a; color=color)
+    end
+end
+
+function print_with_compare(io::IO, a, b, color)
+    if a === b
+        print(io, a)
+    else
+        printstyled(io, a; color=color)
+    end
+end
+
+function show_convert_error(io::IO, ex::MethodError, arg_types_param)
+    # See #13033
+    T = striptype(ex.args[1])
+    if T === nothing
+        print(io, "First argument to `convert` must be a Type, got ", ex.args[1])
+    else
+        printstyled(io, "Cannot `convert` an object of type\n ")
+        print_with_compare(io, arg_types_param[2], T, :green)
```

Comment:
```suggestion
        print_with_compare(io, arg_types_param[2], T, :light_green)
```

to match with the green in the julia prompt